reflex-algos/src/main/scala/org/mlpiper/infrastructure/factory/PythonComponent.scala

```diff
@@ -70,8 +70,8 @@ class PythonComponent(componentMetadata: ComponentMetadata, componentDir: String
 
 object PythonComponent {
   def getAsPythonComponent(comp: ReflexPipelineComponent): PythonComponent = {
-    require(comp.engineType == ComputeEngineType.Generic ||
-      comp.engineType == ComputeEngineType.RestModelServing,
+    require(comp.engineType == ComputeEngineType.Generic || comp.engineType == ComputeEngineType.RestModelServing
```

Comment:
looks like it is time to replace all case with a list:
require(comp.engineType in [a,b,c])

Reply:
This is scala, and therefore no such idiom. We could build a set and use the `contains` function, but I'm not sure it worth it.

Reply:
oops, sorry